en/docs/deploy/migrate/migrate-to-600.md

```diff
@@ -122,7 +122,7 @@ If you have created secondary user stores in the previous WSO2 IS version, copy
 
 ### Webapps
 
-If you have deployed custom webapps in the previous WSO2 Identity Server, update the webapps to be compatible with WSO2 IS 6.0.0 and copy the webapps to the `<NEW_IS_HOME>/repository/deployment/server/webapps` folder. 
+If you have deployed custom webapps in the previous WSO2 Identity Server, update the webapps to be compatible with WSO2 IS 6.0.0 and copy those changes manually to the relevant files in the`<NEW_IS_HOME>/repository/deployment/server/webapps` folder with careful inspection. 
```

Comment:
I think the content here is a bit jumbled. It should look like the following.

![image](https://user-images.githubusercontent.com/33062368/195568647-d1a84ea0-f110-4f88-9924-d758e3c2c51f.png)

If there are custom webapps in the previous IS, those should be made compatible with the newer IS. If the existing webapps were customized in the previous IS, then those changes should be added to the relevant files in the new IS with careful inspection.